src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -874,7 +874,7 @@ await builder.WithClientCredentialsAsync(
             {
                 Logger.TokenAcquisitionError(
                     _logger,
-                    IDWebErrorMessage.ExceptionAcquiringTokenForConfidentialClient,
+                    IDWebErrorMessage.ExceptionAcquiringTokenForConfidentialClient + ex.Message,
```

Comment:
Will this double log the exception? The exception is passed into the logger as well.

Reply:
No, the logger logs the error message passed here. See https://github.com/AzureAD/microsoft-identity-web/blob/master/src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.Logger.cs#L17